db/migrate/20160211194435_create_locations.rb

```diff
@@ -0,0 +1,13 @@
+class CreateLocations < ActiveRecord::Migration
+  def change
+    create_table :locations do |t|
+      t.integer :location_id, null: false
+      t.integer :area_id, null: false
```

Comment:
Not that I know of.  I used what they were called in Grossman, e.g. InvPallet_MainInfo.  I could leave of the _id.